src/test/scala/scalafix/internal/sbt/SbtCompletionsSuite.scala

```diff
@@ -32,8 +32,14 @@ class SbtCompletionsSuite extends FunSuite {
   }
   git.tag("v0.1.0")
 
+
+  val exampleDependency = sbt.ModuleID("com.geirsson", "example-scalafix-rule_2.12", "1.2.0")
+  val (_, mainArgs) = scalafix.sbt.ScalafixPlugin.classloadScalafixAPI(ConsoleLogger(), Seq(exampleDependency))
+  val loadedRules = mainArgs.availableRules.asScala.toList
+  loadedRules.foreach(println)
```

Comment:
This only show built-in rules:

```
ScalafixRule(DisableSyntax)
ScalafixRule(ExplicitResultTypes)
ScalafixRule(NoAutoTupling)
ScalafixRule(NoValInForComprehension)
ScalafixRule(ProcedureSyntax)
ScalafixRule(RemoveUnusedImports)
ScalafixRule(RemoveUnusedTerms)
ScalafixRule(LeakingImplicitClassVal)
ScalafixRule(SyntacticRule)
ScalafixRule(SemanticRule)
```

Reply:
These two rules here are from example-scalafix-rule:

> ScalafixRule(SyntacticRule)
> ScalafixRule(SemanticRule)

Seems to work as expected.

Reply:
Oh I see: https://github.com/olafurpg/example-scalafix-rule/blob/master/scalafix/rules/src/main/scala/fix/Examplescalafixrule_v1.scala

It's not picking up v0 however. I guess this is expected as well.

Reply:
That's expected, only v1 rules can use the service loader framework.